libdeno/deno.h

```diff
@@ -60,10 +60,6 @@ int deno_execute(Deno* d, void* user_data, const char* js_filename,
 // If this is called after deno_recv_cb has returned, the deno_respond
 // will call into the JS callback specified by libdeno.recv().
 //
-// (Ideally, but not currently: After calling deno_respond(), the caller no
-// longer owns `buf` and must not use it; deno_respond() is responsible for
-// releasing its memory.)
-//
```

Comment:
Why remove this comment? This is still the aim - unless I missed something we're still memcpy'ing the response.